### PR TITLE
Oozie script repository and sharedlib 

### PR DESCRIPTION
**Pull request by trK54Ylmz**
Hi,

I've made few changes about bigtop repository and sharedlib also tested on `1.1.15`.

I believe restarting to necessary services would be enough because `reboot` does not working for me.

#85 directly relevant with this PR.

### CODE REVIEW

**Comment by trK54Ylmz** [comment]
I signed it!

File: oozie/oozie.sh [hunk 1]
```diff
@@ -56,15 +65,31 @@ if [[ "${ROLE}" == 'Master' ]]; then
...
+  service hadoop-mapreduce-historyserver restart
+  service hadoop-yarn-resourcemanager restart
+  service hive-server2 restart
+  service oozie start
+
+  # Create sharedlib
+  oozie-setup sharelib create -fs hdfs://${HOSTNAME}
+
+  # Update available sharedlist
+  oozie admin -sharelibupdate -oozie http://localhost:11000/oozie
```
Comment (trK54Ylmz) [inline oozie/oozie.sh]: This could be `${HOSTNAME}` but works both ways.

```
oozie admin -sharelibupdate -oozie http://${HOSTNAME}:11000/oozie
```

**Comment by nloadholtes** [comment]
I used this PR and it worked for me. Is it going to be merged into master?

**Comment by dennishuo** [comment]
@trK54Ylmz Thanks for sending this pull request! And sorry it required such workarounds to get it working; we're still working on longer-term more official support in future versions of the bigtop-based distro so that ideally it'd work out of the box and wouldn't require pinning distro versions and having workaround for any significant period of time, but depending on timelines, I suppose it could still be worth having it in as a temporary working solution until we get it cleaned up.

I'll try to find some cycles from our team to assess a bit more and determine how best to proceed.

**Comment by dossett** [comment]
@trK54Ylmz Nice script!  The command `oozie-setup sharelib create -fs hdfs:` fails with this error:

```
org.apache.oozie.service.HadoopAccessorException: E0902: Exception occured: [No FileSystem for scheme: hdfs]
	at org.apache.oozie.service.HadoopAccessorService.createFileSystem(HadoopAccessorService.java:496)
	at org.apache.oozie.tools.OozieSharelibCLI.run(OozieSharelibCLI.java:159)
	at org.apache.oozie.tools.OozieSharelibCLI.main(OozieSharelibCLI.java:57)
```

Have you run into that problem?

**Comment by dossett** [comment]
The issue was interaction between oozie 4.2.0 and hadoop 2.7.3  I had to:

1) add this to core-site:

```xml
<property>
<name>fs.hdfs.impl</name>
<value>org.apache.hadoop.hdfs.DistributedFileSystem</value>
</property>
```

2) edit the oozie-setup script to include `/usr/lib/hadoop/client/*`in the classpath when it launched the oozie setup CLI tool

**Comment by medb** [comment]
This workaround seems to be obsolete.

**Pull request closed without merging**